csharp-code-evaluator-ut/CSharpCodeEvaluatorTestClass/CSharpCodeEvaluatorTestClass.csproj

```diff
@@ -37,6 +37,10 @@
     <AssemblyOriginatorKeyFile>CSharpCodeEvaluatorTestClass.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="oracle.dataaccess, Version=4.112.4.0, Culture=neutral, PublicKeyToken=89b483f429c47342, processorArchitecture=AMD64">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>C:\Windows\Microsoft.NET\assembly\GAC_64\Oracle.DataAccess\v4.0_4.112.4.0__89b483f429c47342\oracle.dataaccess.dll</HintPath>
```

Comment:
@yyang-convey , you should change this putting the odp.net library on the dependencies folder, as we did with NUnit. You could in fact have a subfolder for x64 and another one for x86